File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/iget-invest-vetting-batch_result.ts



import { InvestVettingBatch_Item } from './invest-vetting-batch_item';

export interface IGetInvestVettingBatch_Result {
  
	InvestVettingBatchItems?: InvestVettingBatch_Item[];
	FileVersion: number;
	FileName: string;
	FileSize: number;
	FileHash: number[];
	FileContent: number[];

}

File: GTTS/INL.VettingService.Models/IGetInvestVettingBatch_Result.cs
using INL.Services.Models;
using System.Collections.Generic;

namespace INL.VettingService.Models
{
    public interface IGetInvestVettingBatch_Result : IBaseResult
    {
        List<InvestVettingBatch_Item> InvestVettingBatchItems { get; set; }
        int FileVersion { get; set; }
        string FileName { get; set; }
        int FileSize { get; set; }
        byte[] FileHash { get; set; }
        byte[] FileContent { get; set; }
    }
}
